### ISSUE TITLE
Cursor issue

### ISSUE THREAD

**Issue by vedran-milic** [tom-s/react-drag-sortable#15]
Hi. Cool library. I have an issue with clickable elements inside a draggable container. When I click on that element cursor changes to draggable cursor, and stays that way until I click on draggable element again or start and stop drag. 
Is there a way to stop this behavior?

**Comment by tom-s** [comment]
Hi. I haven't seen that behavior. Which browser do you get it on ? Also do you have the same issue with the demo page (if not please provide a sample of code) ?

**Comment by vedran-milic** [comment]
This is dumbed down version of the code. Issue is present, I've tested it out.
```javascript
import React, { Component } from 'react';
import Isvg from 'react-inlinesvg';
import iconZvezda from '../../assets/svg/zvezda.svg';
import iconClose from '../../assets/svg/x.svg';
import DragSortableList from 'react-drag-sortable';

class GitTest extends Component {
  constructor(props) {
    super(props);
    this.onSort = this.onSort.bind(this);
    this.setPrimaryImage = this.setPrimaryImage.bind(this);
    this.removeImage = this.removeImage.bind(this);
    this.state = {
      files: [],
      images: [],
      primaryImage: 0
    }
  }

  setPrimaryImage(index) {
    console.log('Clicked on set primary button');
  }

  removeImage(id, idx) {
    console.log('Clicked on delete button');
  }

  onSort(sortedList) {
    console.log("sortedList", sortedList);
  }

  render() {
    let listGrid = [
      {content: (
        <div className="image">
          <div className="image-wrap">
            <a onClick={this.setPrimaryImage}
                className={'set-as-primary no-drag'}><Isvg src={iconZvezda}/></a>
            <img src='image-src' alt='image-alt'/>
          </div>
            <a className="delete-image no-drag" onClick={this.removeImage}><Isvg src={iconClose}/></a>
        </div>
      )},
      {content: (
        <div className="image">
          <div className="image-wrap">
            <a onClick={this.setPrimaryImage}
              className={'set-as-primary no-drag'}><Isvg src={iconZvezda}/></a>
            <img src='image-src' alt='image-alt'/>
          </div>
          <a className="delete-image no-drag" onClick={this.removeImage}><Isvg src={iconClose}/></a>
        </div>
      )},
      {content: (
        <div className="image">
          <div className="image-wrap">
            <a onClick={this.setPrimaryImage}
              className={'set-as-primary no-drag'}><Isvg src={iconZvezda}/></a>
            <img src='image-src' alt='image-alt'/>
          </div>
          <a className="delete-image no-drag" onClick={this.removeImage}><Isvg src={iconClose}/></a>
        </div>
      )},
    ];

    return(
      <div className="objava-osnovni-podaci">
        <DragSortableList items={listGrid} dropBackTransitionDuration={0.3} onSort={this.onSort} type="grid"/>
      </div>
    );
  }
}

export default GitTest;
```

**Comment by tom-s** [comment]
What browser do you see the issue on ?

**Comment by vedran-milic** [comment]
Oh. Sry. It's latest version of chrome. Just click on one button inside draggable container and move cursor.

**Comment by vedran-milic** [comment]
On demo page http://experiments.thomschell.com/react-drag-sortable/demo/build/ I don't have an issue it works great. Only when I have clickable element inside draggable element like <code>button</code>. When I click on button, cursor changes to draggable cursor and stays that way until I click again, or until I do drag again.

**Comment by larsblumberg** [comment]
@tom-s I'm having the same issue on http://experiments.thomschell.com/react-drag-sortable/demo/build/

Steps to reproduce:
1. On latest macOS/Chrome simply click (!) a draggable item without dragging it
2. Overall cursor changes: `<html lang="en" style="cursor: move;">`
3. Cursor remains forever

That's a show stopper for me as my items in the list are supposed to be clickable, too.

**Comment by tom-s** [comment]
Thanks for the precise steps, I managed to reproduce it as well. I'll fix it when I have a bit of time (most likely this week end)

**Comment by tom-s** [comment]
Hi I've published version 1.0.5 which fixes the issue. It was due to interact.js automatically handling cursors, I've turned this option off. You'll need to add you own css if you want to change the cursor.
Let me know if it works for you !

**Comment by vedran-milic** [comment]
Like a charm. Thanks 👍
